src/main/java/com/tstepnik/planner/service/UserService.java

```diff
@@ -25,13 +25,14 @@ public UserService(UserRepository userRepository) {
         return userRepository.findAll();
     }
 
+    //TODO change logic, don;t use database for logged user
     public User getLoggedUser(Principal principal) {
         String login = principal.getName();
         return userRepository.findByLogin(login).orElseThrow(() -> new EntityNotFoundException("User not found"));
     }
 
     public User updateUser(User user, long id) {
-        Optional<User> currentUser = userRepository.findById(id);
+        Optional<User> currentUser = userRepository.findById(id);//TODO change logic, example: .orElseTrow...
```

Comment:
todo?

Reply:
Yes, i don't want change class which is not in my feauture branch  so i just added TODO to remember i should do sth there.

Reply:
got it.